src/components/Todo/dist/Todo.js

```diff
@@ -0,0 +1,177 @@
+"use strict";
```

Comment:
Do not push anything from the dist folder. I am adding dist folder inside .gitignore please that a pull